controllers/twist_controller/src/twist_controller.cpp

```diff
@@ -0,0 +1,129 @@
+// Copyright 2023 Kristófi Mihály
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "twist_controller.hpp"
+
+namespace kuka_controllers
+{
+controller_interface::CallbackReturn TwistController::on_init()
+{
+  return controller_interface::CallbackReturn::SUCCESS;
+}
+
+controller_interface::InterfaceConfiguration TwistController::
+command_interface_configuration()
+const
+{
+  controller_interface::InterfaceConfiguration config;
+  config.type = controller_interface::interface_configuration_type::INDIVIDUAL;
+  // config.names.emplace_back("twist commands");
+  config.names.emplace_back("twist/linear/x");
+  config.names.emplace_back("twist/linear/y");
+  config.names.emplace_back("twist/linear/z");
+  config.names.emplace_back("twist/angular/x");
+  config.names.emplace_back("twist/angular/y");
+  config.names.emplace_back("twist/angular/z");
+  return config;
+}
+
+controller_interface::InterfaceConfiguration TwistController::state_interface_configuration()
+const
+{
+  controller_interface::InterfaceConfiguration config;
+  config.type = controller_interface::interface_configuration_type::INDIVIDUAL;
+  config.names.emplace_back("twist/linear/x");
```

Comment:
these are not used at all, so should not be specified here
a controller should only claim the interfaces it actually uses, it is not a problem, that some interfaces remain unclaimed